teams/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,30 @@
+# do odpalania lokalnie poza docker compose
+spring:
+    datasource:
+        url: jdbc:postgresql://localhost:5435/teams
+        username: postgres
+        password: postgres
+        driver-class-name: org.postgresql.Driver
+    jpa:
+        show-sql: true
+        hibernate:
+            ddl-auto: create-drop
+        properties:
+            hibernate:
+                default_schema: public
+        database: postgresql
+        database-platform: org.hibernate.dialect.PostgreSQLDialect
+    jackson:
+        deserialization:
+            fail-on-unknown-properties: false
+    security:
+        oauth2:
+            resourceserver:
+                jwt:
+                    issuer-uri: http://keycloak:8443/realms/lolmatch
+    rabbitmq:
+        addresses: localhost
+        username: guest
+        password: guest
+server:
+    port: 8090
```

Comment:
nie będzie to potrzebne w innych propertisach?

Reply:
chyba nie, jak będzie to dodamy